kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -90,7 +90,7 @@ guDateRange.directive('guDateRange', [function () {
             });
 
             $scope.$watch('pikaStartValue', function (pikaStartValue) {
-                var date = pikaStartValue && new Date(pikaStartValue);
+                var date = (pikaStartValue && new Date(pikaStartValue)) || new Date();
```

Comment:
Is this causing an error? Have you tested that the behaviour is still as expected?

cc @akash1810 who worked on that bit.